core/src/mindustry/editor/EditorTool.java

```diff
@@ -1,15 +1,22 @@
 package mindustry.editor;
 
 import arc.func.*;
+import arc.maps.TileSet;
 import arc.math.*;
 import arc.math.geom.*;
 import arc.struct.*;
 import arc.util.*;
 import mindustry.content.*;
+import mindustry.entities.type.TileEntity;
 import mindustry.game.*;
+import mindustry.maps.Map;
 import mindustry.world.*;
 import mindustry.world.blocks.*;
 
+import java.util.ArrayList;
```

Comment:
Read [CONTRIBUTING.MD](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md). Do not use standard Java collections.